lib/fluent/plugin/out_rdkafka2.rb

```diff
@@ -270,6 +270,8 @@ def build_config
       config[:"queue.buffering.max.messages"] = @rdkafka_buffering_max_messages if @rdkafka_buffering_max_messages
       config[:"message.max.bytes"] = @rdkafka_message_max_bytes if @rdkafka_message_max_bytes
       config[:"batch.num.messages"] = @rdkafka_message_max_num if @rdkafka_message_max_num
+      config[:"sasl.username"] = @username if @username
+      config[:"sasl.password"] = @password if @password
```

Comment:
> This commit makes it such that we are able to leverage the existing
> password parameter that is already marked as secret.

Is it correct? out_rdkafka2 doesn't seem have these parameters.

Reply:
Ah yeah, if I'm understanding the code properly, we're including SASL related configuration on line 104 of out_rdkafka2, which should have username and password defined

Reply:
Sorry, I missed it.